packages/trader/src/AppV2/Components/TradeParameters/PayoutPerPoint/payout-per-point-wheel.tsx

```diff
@@ -1,87 +1,118 @@
 import React from 'react';
-import debounce from 'lodash.debounce';
+import { observer } from 'mobx-react';
+import { useTraderStore } from 'Stores/useTraderStores';
 import { ActionSheet, Text, WheelPicker } from '@deriv-com/quill-ui';
 import { Skeleton } from '@deriv/components';
 import { Localize } from '@deriv/translations';
-import type { TV2ParamsInitialValues } from 'Stores/Modules/Trading/trade-store';
+import { getProposalRequestObject } from 'AppV2/Utils/trade-params-utils';
+import { useDtraderQuery } from 'AppV2/Hooks/useDtraderQuery';
+import { TTradeStore } from 'Types';
 
 type TPayoutPerPointWheelProps = {
     barrier?: string | number;
+    is_open?: boolean;
     current_payout_per_point: string;
     onPayoutPerPointSelect: (new_value: string | number) => void;
+    onClose: () => void;
     payout_per_point_list: {
         value: string;
     }[];
-    setV2ParamsInitialValues: ({ value, name }: { value: number | string; name: keyof TV2ParamsInitialValues }) => void;
 };
+type TOnProposalResponse = TTradeStore['onProposalResponse'];
 
-const onWheelPickerScrollDebounced = debounce(
-    (new_value: string | number, callback: TPayoutPerPointWheelProps['onPayoutPerPointSelect']) => callback(new_value),
-    200
-);
-
-const PayoutPerPointWheel = ({
-    barrier,
-    current_payout_per_point,
-    onPayoutPerPointSelect,
-    setV2ParamsInitialValues,
-    payout_per_point_list,
-}: TPayoutPerPointWheelProps) => {
-    const initial_value_ref = React.useRef<string | number>();
-    const selected_value_ref = React.useRef<string | number>(current_payout_per_point);
+const PayoutPerPointWheel = observer(
```

Comment:
The reason of the bug is that we were using payout_per_point values from local storage and they were  overwriting the correcnt ones when user was swathing the symbols. I removed all logic dedicated to the local storage.